daml-script/test/BUILD.bazel

```diff
@@ -157,6 +159,10 @@ da_scala_test_suite(
         ":script-test-1.dev.dar",
         ":script-test-no-ledger.dar",
         "//ledger/test-common/test-certificates",
+        "@openssl_dev_env//:bin/openssl" if not is_windows else "@openssl_dev_env//:usr/bin/openssl.exe",
```

Comment:
Perhaps pull this out into a variable?

Reply:
Do you mean a local variable in the same build file?

Reply:
Yup.

```bazel
openssl = "@openssl_dev_env//:bin/openssl" if not is_windows else "@openssl_dev_env//:usr/bin/openssl.exe"
```